constants/version.go

```diff
@@ -3,3 +3,7 @@ package constants
 const VersionDefault = "Piped into LDflags on build. You are probably running Railway CLI from source."
 
 var Version string = VersionDefault
+
+func IsDevVersion() bool {
```

Comment:
Swear to get we have this or an equivalent somewhere, but I might have removed it....